build.sh

```diff
@@ -12,18 +12,21 @@ shift "$(( OPTIND - 1 ))"
 
 NAME=$(echo "${PWD##*/}" | tr _ -)
 TAG="latest"
+MULTISTAGE_TARGET="development"
```

Comment:
I would propose it like that. We don't know yet how we are going to use it so its useless to create several scripts or so but if we put it there, its clear that this option exists

Reply:
I advise you append the image name tag with the build stage target (slash separated, so for example it becomes `control-libraries/development`)

Reply:
@buschbapti what do you think about appending the target name to the image tag?

Reply:
i think that would be reasonable